develop-docs/application/dynamic-sampling/extrapolation.mdx

```diff
@@ -0,0 +1,95 @@
+
+- Accuracy refers to data being correct. For example, the measured number of spans corresponds to the actual number of spans that were executed. As sample rates decrease, accuracy also goes down, because minute random decisions can influence the result in major ways, in absolute numbers. Also, when traffic is low and 100% of data is sampled, the system is fully accurate despite aggregates being affected by inherent statistical uncertainty.
+- Expressiveness refers to data being able to express something about the state of the observed system. For example, a single sample with specific tags and a full trace can be very expressive, and a large amount of spans can have very misleading characteristics. Expressiveness therefore depends on the use case for the data.
+
+Affected Product Surface
+
+1. Explore
+2. Alerts
+3. Dashboards
+4. Requests
+5. Profiles
+
+### **Modes**
+
+There are two modes that can be used to view data in Sentry: extrapolated mode and sample mode.
+
+- Extrapolated mode extrapolates the ingested data as outlined above.
+- Sample mode does not extrapolate and presents exactly the data that was ingested.
+
+Depending on the context and the use case, one mode may be more useful than the other. 
+
+There is currently no way for Sentry to automatically switch from the extrapolated mode into sample mode based on query attributes, therefore the transition needs to be triggered by the user. However, Sentry can nudge the user, based on observed characteristics of a query, to switch from one mode to another. One example for this is when an ID column is detected: extrapolated aggregates for high-cardinality and low-volume ID columns are usually not very useful, because they may refer to a highly exaggerated volume of data that is not extrapolated correctly due to the high-cardinality nature of the column in question.
+
+## Aggregates
+
+Sentry allows the user to aggregate data in different ways - the following aggregates are generally available, along with whether they are extrapolatable or not:
+
+| **Aggregate** | **Can be extrapolated?** |
+| --- | --- |
+| mean | yes |
+| min | no |
+| count | yes |
+| sum | yes |
+| max | no |
+| percentiles | yes |
+| count_unique | no |
+
+Each of these aggregates has their own way of dealing with extrapolation, due to the fact that e.g. counts have to be extrapolated in a slightly different way from percentiles. 
+
+[Insert text about how different extrapolation mechanisms work]
+
+As long as there are sufficient samples, the sample rate itself does not matter as much, but due to the extrapolation mechanism, what would be a fluctuation of a few samples, may turn into a much larger absolute impact e.g. in terms of the view count. Of course, when a site gets billions of visits, a fluctation of 100.000 via the noise introduced by a sample rate of 0.00001 is not as salient. 
+
+## Why do we even extrapolate?
```

Comment:
Personally, this section feels a bit out of place and slightly redundant with the intro. What if we merge this?

Reply:
Great catch!